For Advanced/040.md

```diff
@@ -0,0 +1,22 @@
+# Message in a Robot
```

Comment:
I dont know how titles are chosen but imho title
```
Message in a Robot
``` 
does not suggest email. Anyone looking for emails control in robot will have difficult time finding answers

Reply:
these lessons have titles that are more fun than information. Perhaps I should give leads or other headers with a short description of a task, eg. "Email handling"